subcommand_plugin_examples/rust_subcommand_plugin_tutorial/README.md

```diff
@@ -0,0 +1,231 @@
+[template]: https://github.com/conda/conda-plugin-template/subcommand_plugin_examples/rust_subcommand_plugin_tutorial/generate
+[pyproject.toml docs]: https://packaging.python.org/en/latest/tutorials/packaging-projects/#creating-pyproject-toml
+[entrypoints docs]: https://packaging.python.org/en/latest/specifications/entry-points/
+[licenses]: https://docs.conda.io/projects/conda/en/latest/dev-guide/plugin-api/index.html#a-note-on-licensing
+[pep 621]: https://peps.python.org/pep-0621/
+[maturin]: https://github.com/PyO3/maturin
+[pyo3]: https://github.com/PyO3/pyo3
+[pluggy docs]: https://pluggy.readthedocs.io/en/stable/index.html
+
+# Custom conda Subcommand Plugin Tutorial Written in Rust
+
+In this tutorial, we will create a new conda subcommand written in Rust that multiplies two integers.
+
+To follow along with this guide, make sure you have the latest conda and conda-build installed:
+
+```bash
+(base) $ conda update conda conda-build
```

Comment:
```suggestion
$ conda update conda conda-build pip
```

Same comment as above. Let's just keep this as simple as possible and not make assumptions about which environment this is in.